cmake/modules/HexagonSDK.cmake

```diff
@@ -97,11 +132,26 @@ function(_get_hexagon_sdk_property_impl
     endif()
 
   elseif(_property STREQUAL "QAIC_EXE")
-    _check_path_exists(
-      "${_hexagon_sdk_root}/ipc/fastrpc/qaic/Ubuntu18/qaic"
-      _qaic_path
-    )
-    set_parent(${_output_variable} "${_qaic_path}")
+    set(_override $ENV{QAIC_PATH_OVERRIDE})
```

Comment:
Would it be more idiomatic for `QAIC_PATH_OVERRIDE` to be a CMake variable, rather than an environment variable?

Reply:
The motivation for an environment variable is that one can set it and then use cmake as if nothing happened, so to speak.  Since cmake calls itself recursively when building various projects, passing of a cmake variable would need to be explicit.

Reply:
Thanks, that makes sense.